addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -562,6 +576,14 @@ describe('List', () => {
                             `),
                         });
                     });
+                    it('should create a new checked list if closestBlock is deeply nested inside a nav-item list', async () => {
```

Comment:
And here.

And I would add a case for each of these, where it should outdent (`ul > li.nav-item > div > ul > li > p` for instance).